### DISCUSSION TITLE
JSON File option for custom list of BANDITs created

### DISCUSSION THREAD

**Discussion by wifiman06060** [nzymedefense/nzyme#754] [Ideas]
I have multiple sensors running nzyme.  Would be very useful to be able to export a custom BANDITs list that i create in one sensor to multiple ones.  A JSON file that I can export and then import to other sensors would do the trick.

**Comment by lennartkoopmann** [comment]
Hi, thanks for the feature request! This will work with the v2.0 multi-node architecture out of the box and without the need to export/import because all nodes will share configuration.
